python/mxnet/module/base_module.py

```diff
@@ -64,16 +64,16 @@ def _check_names_match(data_names, data_shapes, name, throw):
             warnings.warn(msg)
 
 
-def _parse_data_desc(data_names, label_names, data_shapes, label_shapes):
```

Comment:
argument name changes has no practical benefit but can cause problems. Let's keep the original names and improve documents.